src/translation/translation_pl_PL.ts

```diff
@@ -3724,7 +3724,7 @@ Twój kanał został wyciszony i włączono „Wycisz mnie”. Napraw przyczynę
     <message>
         <location filename="../serverdlgbase.ui" line="75"/>
         <source>Directory</source>
-        <translation type="unfinished">Serwer</translation>
+        <translation>Serwer</translation>
```

Comment:
Is Serwer = Server or Directory?

It should somewhat fit to "Adres własnego serwera zbiorczego"

Reply:
It is, believe me :-)

Reply:
Great ;-).